docs/docs/rasa-pro-changelog.mdx

```diff
@@ -0,0 +1,95 @@
+---
+id: rasa-pro-changelog
+sidebar_label: Rasa Pro Change Log
+title: Rasa Pro Change Log
+---
+
+All notable changes to Rasa Pro will be documented in this page.
+This product adheres to [Semantic Versioning](https://semver.org/) starting with version 3.3 (initial version).
+
+Rasa Pro consists of two deployable artifacts: Rasa Plus and Rasa Pro Services. You can read the change log for both artifacts below.
+
+## Rasa Pro 3.6
+
+### Rasa Plus 3.6
+
+#### Features
+- Implemented PII (Personally Idenfiable Information) management using Microsoft Presidio as the entity analyzer and
+  anonymization engine.
+  The feature covers the following:
+  - anonymization of Rasa events (`UserUttered`, `BotUttered`, `SlotSet`, `EntitiesAdded`) before they are streamed to
+  Kafka event broker anonymization topics specified in `endpoints.yml`.
+  - anonymization of Rasa logs that expose PII data
+
+  The main components of the feature are:
+  - anonymization rules that define in `endpoints.yml` the PII entities to be anonymized and the anonymization method to be used
+  - anonymization executor that executes the anonymization rules on a given text
+  - anonymization orchestrator that orchestrates the execution of the anonymization rules and publishes
+  the anonymized event to the matched Kafka topic.
+  - anonymization pipeline that contains a list of orchestrators and is registered to a singleton provider component,
+  which gets invoked in hook calls in Rasa Open Source when the pipeline must be retrieved for anonymizing events and logs.
+
+  Please read through the [PII Management](./pii-management.mdx) section in the official documentation to learn how to get started.
+
+- Implemented support for real time evaluation of [Markers](./monitoring/analytics/realtime-markers.mdx) with the Analytics
+  Data Pipeline. For this feature, we've added support for `rasa markers upload` command. Running this command validates the marker configuration file against the domain file and uploads the configuration to Analytics Data Pipeline.
+
+### Improvements
+- Add `rasa marker upload` command to upload markers to the Rasa Pro Services.
+  Usage: `rasa marker upload --config=<path-to-config-file> -d=<path-to-domain-file> -rasa-pro-services-url=<url>`.
+- Enhance the validation of the `anonymization` key in `endpoints.yaml` by introducing checks for required fields and duplicate IDs.
+
+### Bugfixes
+- Anonymize `rasa_events` structlog key.
+- Fixes issue with uploading locally trained model to a cloud rasa-plus instance where the conversation does not
+  go as expected because slots don't get set correctly, e.g. an error is logged `Tried to set non existent slot 'placeholder_slot_name'. Make sure you added all your slots to your domain file.`.
```

Comment:
```suggestion
  go as expected because slots don't get set correctly, e.g. an error is logged "Tried to set non existent slot 'placeholder_slot_name'." Make sure you added all your slots to your domain file.
```